features/content-visibility.yml.dist

```diff
@@ -2,16 +2,13 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
-  baseline_low_date: 2024-09-16
+  baseline: false
```

Comment:
Since we're regressing the status, would you add a comment to the authored yaml file with a note as in https://github.com/web-platform-dx/web-features/pull/1973?

Reply:
This should no longer be a regression with https://github.com/mdn/browser-compat-data/pull/25186 in 5.6.19.

Reply:
There are now no status changes here.